website/integrations/sources/active-directory/index.md

```diff
@@ -31,17 +31,17 @@ The following placeholders will be used:
 
     ![](./03_additional_perms.png)
 
-Additional infos: https://support.microfocus.com/kb/doc.php?id=7023371
+Additional info: https://support.microfocus.com/kb/doc.php?id=7023371
 
 ## authentik Setup
 
-In authentik, create a new LDAP Source in Resources -> Sources.
+In authentik, create a new LDAP Source in Directory -> Federation & Social login.
 
 Use these settings:
 
 - Server URI: `ldap://ad.company`
 
-    For authentik to be able to write passwords back to Active Directory, make sure to use `ldaps://`
+    For authentik to be able to write passwords back to Active Directory, make sure to use `ldaps://`. You can test to verify LDAPS is working using `ldp.exe`.
```

Comment:
Maybe add a link for https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-r2-and-2012/cc771022(v=ws.11) instead of just mentioning ldp

Reply:
That doesn't really explain how to test either. I figure leave it up to the reader.